frontend/app/src/shared/components/inputs/dropdown.tsx

```diff
@@ -151,6 +153,8 @@ export const DropdownAddAction: React.FC<DropdownAddActionProps> = ({
       <Button
         className="w-full bg-custom-blue-700/10 border border-custom-blue-700/20 text-custom-blue-700 enabled:hover:bg-custom-blue-700/20"
         onClick={() => setOpen(!open)}
+        disabled={!namespace?.user_editable}
```

Comment:
How about hiding it instead? Since the button can’t be used in this case, disabling it might be confusing or suggest that it could work under certain conditions.
I suggest hiding it to avoid false expectations.